app.rb

```diff
@@ -0,0 +1,116 @@
+
+# Addition function
+def add_op
+  $stdout.puts "Number 1?"
+  $stdout.flush
+  num1 = $stdin.gets.chomp.to_i
+  $stdout.puts "Number 2?"
+  $stdout.flush
+  num2 = $stdin.gets.chomp.to_i
+  puts num1 + num2
+end
+
+# Subtraction function
+def sub_op
+  $stdout.puts "Number 1?"
+  $stdout.flush
+  num1 = $stdin.gets.chomp.to_i
+  $stdout.puts "Number 2?"
+  $stdout.flush
+  num2 = $stdin.gets.chomp.to_i
+  puts num1 - num2
+end
+
+# Multiplication function
+def mul_op
+  $stdout.puts "Number 1?"
+  $stdout.flush
+  num1 = $stdin.gets.chomp.to_i
+  $stdout.puts "Number 2?"
+  $stdout.flush
+  num2 = $stdin.gets.chomp.to_i
+  puts num1 * num2
+end
+
+# Division function
+def div_op
+  $stdout.puts "Number 1?"
+  $stdout.flush
+  num1 = $stdin.gets.chomp.to_i
+  $stdout.puts "Number 2?"
+  $stdout.flush
+  num2 = $stdin.gets.chomp.to_i
+  puts num1 / num2
```

Comment:
It's not working properly for any division with a reminder (for example try to divide 57 by 5)